lisa/base_tools/service.py

```diff
@@ -67,6 +67,9 @@ def is_service_inactive(self, name: str) -> bool:
     def is_service_running(self, name: str) -> bool:
         return self._internal_tool._check_service_running(name)  # type: ignore
 
+    def wait_for_service_start(self, name: str) -> None:
+        pass
```

Comment:
Raise Not implement error, so we know the code runs into an unknown path.

Reply:
Fixed